jobs/furnishings/src/furnishings/stage_processors/stage_one.py

```diff
@@ -163,7 +164,7 @@ def process_paper_letters(self):
                 with self._bcmail_sftp_connection as client:
                     resp = self.upload_to_sftp(client, self._xpro_letters, filename)
                 self.update_notes_and_status(
-                    self._bc_mail_furnishings,
+                    self._xpro_mail_furnishings,
```

Comment:
Oh my God I can't believe I made this mistake haha. Nice 👍

Reply:
Haha, it happens to the best of us! Glad we caught it in time(^-^)